calico_node/tests/st/policy/test_namespace.py

```diff
@@ -81,6 +82,9 @@ def setUpClass(cls):
         for host in cls.hosts:
             host.start_calico_node()
 
+        # Sleep for a bit to allow the cluster to connect and BGP to establish.
+        time.sleep(15)
```

Comment:
I don't think we can add this to the master code - we need something that's not going to add a bunch of extra time to the tests, like checking that BGP has established before continuing.

Reply:
ok. I'll just have semaphore keep rerunning this PR to at least see if we're seeing less flakes.